### ISSUE TITLE
Search bar doesn't work.

### ISSUE THREAD

**Issue by johnnymatthews** [drand/website#101]
The search functionality comes from Algolia's open-source [Docssearch](https://docsearch.algolia.com/), but it looks like it's borked somehow.

![image](https://user-images.githubusercontent.com/9611008/128395242-a1a49896-bcfe-4540-9880-eb4ad2a68803.png)

**Comment by johnnymatthews** [comment]
TLDR: the search service we use was trying to use [`beta.drand.love`](https://github.com/algolia/docsearch-configs/blob/master/configs/drand.json#L4) instead of `drand.love`. I've rolled back to using the default search from the framework we use. I'm working with the search service providers to get the good search back to `drand.love`.

Long version: when Drand was in beta, we signed up to Algolia's Docsearch, which is a service Algolia provides open-source projects so they can have an effective search solution for free. The URL used for this service was `beta.drand.love`. Some time in the last month or so, Algolia realised that `beta.drand.love` isn't a thing anymore, and stopped indexing out site (and thus stopped providing search stuff for us). To fix this, all we need to do is re-apply to the Docsearch open-source projects program - but this takes a few days. In the meantime, I've reinstalled the default search that comes with VuePress, the framework we use to build `drand.love`. The default search is _ok_, but it only looks at `h1`, `h2`, and `h3` tags for search terms. Anything within the content of a page is ignored. However, we should have the full-super-mega-good search from Algolia back by the end of this week.
